content/en/tracing/trace_collection/compatibility/java.md

```diff
@@ -279,6 +279,111 @@ Integrations can be enabled or disabled individually (overriding the default abo
 - Running the Java tracer in Bitbucket is not supported.
 - Loading multiple Java Agents that perform APM/tracing functions is not a recommended or supported configuration.
 
+## GraalVM Native Image support
+
+GraalVM Native Image is a technology that allows you to compile Java applications into native executables. The Datadog Java tracer supports GraalVM Native Image. This allows you to compile your applications into native executables while still benefiting from the tracing capabilities offered by the library.
+
+### Requirements
+
+Use the latest versions of:
+
+- [GraalVM][7]
+- [Datadog Java tracer][1]
+
+### Setup
```

Comment:
Can we rename `Setup` to just `GraalVM` and have 3 separate tabs here
- GraalVM
- Quarkus Native
- Spring Native

Reply:
Good idea. I was thinking that too while working on the content. I'll switch the structure to tabs 👍